build.gradle.kts

```diff
@@ -22,3 +44,40 @@ subprojects {
         systemProperties = systemPropertiesMap
     }
 }
+
+tasks.register("mergeDashFiles") {
```

Comment:
I would have prefered to put this into "dash.gradle.kts" to make it "re-usable" but unfortunately this task is only needed by the root project,... maybe it makes sense to think about a convention like naming this like dash.root.gradle.kts or so

Reply:
I agree as soon as it makes sense to extract this one. Currently it is still pretty clean.